packages/js_of_ocaml-compiler/js_of_ocaml-compiler.3.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer:   "dev@ocsigen.org"
+authors:      "Ocsigen team"
+bug-reports:  "https://github.com/ocsigen/js_of_ocaml/issues"
+homepage:     "http://ocsigen.org/js_of_ocaml"
+dev-repo:     "git+https://github.com/ocsigen/js_of_ocaml.git"
+
+name: "js_of_ocaml-compiler"
+
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
+
+depends: [
+  "jbuilder" {build & >= "1.0+beta7"}
+  "cmdliner"
+  "cppo" {>= "1.1.0"}
+]
+
+depopts: [ "ocamlfind" "yojson" ]
```

Comment:
is it necessary to have ocamlfind as a depopt? It would be better to let the `jbuilder` package handle that perhaps.